assets/src/edit-story/components/tablist/styles.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import {
+  Button,
+  BUTTON_VARIANTS,
+  Text,
+  ThemeGlobals,
+  themeHelpers,
+  THEME_CONSTANTS,
+} from '../../../design-system';
+import { PANEL_STATES } from './constants';
+
+export const Tablist = styled.div`
+  width: 308px;
+  background: ${({ theme }) => theme.colors.bg.primary};
+`;
+Tablist.propTypes = {
+  // label to describe purpose of tabs
+  'aria-label': PropTypes.string.isRequired,
+};
+
+export const PanelWrapper = styled.div`
+  width: 100%;
+  background: ${({ theme }) => theme.colors.bg.primary};
+
+  ${({ isExpanded, theme }) =>
+    isExpanded &&
+    css`
+      & > ${TabButton} {
+        ${SmallHeadline} {
+          opacity: 1;
+        }
+
+        &,
+        :hover,
+        :focus {
+          background-color: ${theme.colors.bg.primary};
+        }
+
+        svg {
+          transform: rotate(0);
+        }
+      }
+
+      & > ${TabPanel} {
+        height: 560px;
+        padding: 16px;
+        visibility: visible;
+      }
+    `};
+`;
+
+export const SmallHeadline = styled(Text).attrs({
```

Comment:
reallyyyy minor nit: naming this `SmallHeadline` doesn't make a lot of sense anymore?

Reply:
omg i forgot to change this 🤕 you're right!